data-serving/data-service/src/model/revision-metadata.ts

```diff
@@ -0,0 +1,35 @@
+import { dateSchema } from './date';
+import mongoose from 'mongoose';
+
+export const revisionMetadataSchema = new mongoose.Schema({
+    id: {
+        type: Number,
+        min: 0,
+        validate: {
+            validator: Number.isInteger,
+            message: '{VALUE} is not an integer value',
+        },
+        required: 'Enter a revision id',
+    },
+    moderator: {
+        type: String,
+        required: 'Enter a revision moderator id',
+    },
+    date: {
+        ...dateSchema,
+        required: 'Enter a revision date',
+    },
+    notes: String,
+});
+
+export type RevisionMetadataDocument = mongoose.Document & {
+    id: number;
+    moderator: string;
+    date: Date;
+    notes: string;
+};
+
+export const RevisionMetadata = mongoose.model<RevisionMetadataDocument>(
```

Comment:
that's fair -- can confirm that the test isn't creating those collections (because validate() doesn't save it). I'm actually only using the model in the tests, so why don't I just move it into the test for peace of mind?

that way, only case.ts will actually have a model